lib/fpm/package/cpan.rb

```diff
@@ -309,17 +309,16 @@ def download(metadata, cpan_version=nil)
                  :distribution => distribution,
                  :version => cpan_version)
 
-    # default to latest versionunless we specify one
+    # default to latest version unless we specify one
     if cpan_version.nil?
-      self.version = metadata["version"]
+      self.version = "#{metadata["version"]}"
```

Comment:
It was just belt and braces to ensure self.version was a string before attempting the substitution below.